lib/cocoapods/generate/podfile_generator.rb

```diff
@@ -37,10 +37,22 @@ def podfile_for_spec(spec)
 
           plugin 'cocoapods-generate'
 
-          install! 'cocoapods',
-                   deterministic_uuids: generator.configuration.deterministic_uuids?,
-                   share_schemes_for_development_pods: generator.configuration.share_schemes_for_development_pods?,
-                   warn_for_multiple_pod_sources: generator.configuration.warn_for_multiple_pod_sources?
+
+          options = {
+            deterministic_uuids: generator.configuration.deterministic_uuids?,
+            share_schemes_for_development_pods: generator.configuration.share_schemes_for_development_pods?,
+            warn_for_multiple_pod_sources: generator.configuration.warn_for_multiple_pod_sources?
+          }
+
+          if Pod::Installer::InstallationOptions.all_options.include?('generate_multiple_pod_projects')
```

Comment:
happy to hear better ways to handle this

Reply:
nit: can we extract `install_options` out into a method on `generator`, so then we can just call `install! 'cocoapods', generator.install_options` ?